src/drivers/mct/cime_config/config_compsets.xml

```diff
@@ -40,32 +41,32 @@
 
   <compset>
     <alias>A</alias>
-    <lname>2000_DATM%NYF_SLND_DICE%SSMI_DOCN%DOM_DROF%NYF_SGLC_SWAV</lname>
+    <lname>2000_DATM%NYF_SLND_DICE%SSMI_DOCN%DOM_DROF%NYF_SGLC_SWAV_SIAC</lname>
```

Comment:
These stub components are no longer necessary, remove them?

Reply:
I think at least one of the shared compsets should have it.  Just so we know its there.